cluster/examples/kubernetes/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -108,6 +108,25 @@ spec:
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
         {{ end }}
+        {{ if .EnableVolumeReplicationSidecar }}
+        - name: volume-replication
+          image: {{ .VolumeReplicationImage }}
+          args :
+            - "--metrics-bind-address=0"
+            - "--leader-election-namespace={{ .Namespace }}"
+            - "--drivername={{ .DriverNamePrefix }}rbd.csi.ceph.com"
+            - "--csi-address=$(ADDRESS)"
+            - "--rpc-timeout=150s"
+            - "--health-probe-bind-address=:9998"
+            - "--leader-elect=false"
+          env:
+            - name: ADDRESS
+              value: unix:///csi/csi-provisioner.sock
+          imagePullPolicy: "IfNotPresent"
+          volumeMounts:
+            - name: socket-dir
```

Comment:
I don't see this volume defined.

Reply:
https://github.com/rook/rook/pull/7403/files#diff-94612bfa354b036a2f57b1771f8b73f02d287d6a72e62b3dc21bcb3a0771d7edL184
isn't this the defn?

Reply:
I see it now, thanks